ui/appui-layout-react/src/appui-layout-react/widget-panels/Panel.scss

```diff
@@ -102,7 +102,8 @@
           }
         }
       }
-      > .nz-grip-container > .nz-grip {
+
+      >.nz-grip-container>.nz-grip {
```

Comment:
Do we have Stylelint or Prettier setup that could catch these for us?

Reply:
Not ATM. We have 83 hits for `>.` and 215 hits for `> .` in our `*.scss` files. I'd prefer the variant w/ more spacing.
We could achieve more consistency by setting `scss.format.spaceAroundSelectorSeparator` in workspace settings?

Reply:
Resolving this for now. I've opened https://github.com/iTwin/appui/issues/331 instead